app/assets/stylesheets/application.scss

```diff
@@ -146,3 +158,24 @@ a:hover {
   text-align: inline;
 }
 
+.nameplate {
+  font-size: 1.6em;
+}
+
+.btn {
+  background:none;
+  border:none;
+  padding:0;
+  font: inherit;
+  cursor: pointer;
+  color: $unvisited-color;
+}
+
+.actions input {
+  background:none;
+  border:none;
+  padding:0;
```

Comment:
Properties should be ordered background, border, color, cursor, font, padding<br>Colon after property should be followed by one space